Files/Filesystem/Drives.cs

```diff
@@ -118,29 +110,39 @@ private async Task RefreshUI()
                 // Defer because UI-thread is not ready yet (and DriveItem requires it?)
                 CoreApplication.MainView.Activated += RefreshUI;
             }
+            Logger.Info("RefreshUI() complete");
```

Comment:
I'd probably remove all this logs

Reply:
Should be able to do without them. Easy enough to add back later for troubleshooting should they be needed for testing.